packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,1933 @@
+openapi: 3.0.0
+
+info:
+  title: UpSwyng API v1
+  description: UpSwyng API v1
+  contact:
+    name: Code For Boulder
+    url: 'https://www.codeforboulder.org/'
+  version: '1.0.0'
+
+servers:
+  - url: 'http://localhost:3000/api'
+    description: Docker Compose Server
+
+# Paths are sorted alphabetically
+# variables as part of a path are sorted as last in order.
+
+paths:
+  /alert:
+    post:
+      description: 'API endpoint to create or update an alert.'
+      summary: 'Create alert'
+      security:
+        - requireAdmin: []
+      tags:
+        - alert
+      operationId: alertPost
+      requestBody:
+        description: New Alert Payload.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AlertPost'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/AlertPost'
+      responses:
+        '200':
+          description: New Alert
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  alert:
+                    $ref: '#/components/schemas/AlertData'
+        '400':
+          $ref: '#/components/responses/ErrorInvalidRequest'
+        '401':
+          $ref: '#/components/responses/ErrorAuthentication'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /alert/search:
+    post:
+      description: 'API endpoint to get a list of Alerts. Returns the full alert (TAlertFull with user who created) if the user is a signed-in admin. Otherwise, returns TAlert'
+      summary: 'API endpoint to get a list of Alerts'
+      tags:
+        - alert
+      operationId: alertSearchPost
+      requestBody:
+        description: Parameters to search for alert.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AlertSearch'
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: '#/components/schemas/AlertSearch'
+      responses:
+        '200':
+          description: Alerts.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    type: integer
+                  alerts:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/AlertData'
+        '404':
```

Comment:
Not sure how detailed we want to be, but this can also return a 500 with the type `message` string